lib/pbench/test/functional/server/test_user.py

```diff
@@ -39,14 +39,14 @@ def test_login_fail(self, pbench_server_client: PbenchServerClient):
                 "first_name": "Test",
                 "last_name": "Account",
                 "password": PASSWORD,
-                "email": f"{USERNAME1}@example.com",
+                "email": f"{USERNAME1}@gmail.com",
```

Comment:
This feels slightly icky to me.  (E.g., presumably, eventually, we'll have a functional test which actually tries to use these email addresses....)

Looking at [RFC 6761](https://www.rfc-editor.org/rfc/rfc6761), would `user@test` or `user@localhost` work?  (It's really annoying that `user@example.com` doesn't work again...that's counter to (my interpretation of) the standard.)

Reply:
It's hard to test given that this isn't happening locally. I suppose I could change `requirements.txt` to require the new version, but it just doesn't seem worthwhile in this context.